css-ecommerce-page/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ecommerce Challenge</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans&display=swap" rel="stylesheet">
+  <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+
+    <div class="header">
+      <h1>New Arrivals</h1>
+    </div>
+    <div class="row">
+      <div class="column column-flex">
+        <img src="images/dress1.jpeg">
+      </div>
+      <div class="column column-flex">
+        <img src="images/ysl-bag.jpeg">
+      </div>
+      <div class="column column-flex">
+        <img src="images/key.jpeg">
+      </div>
+      <div class="column column-flex">
+        <img src="images/lv-bag.jpg">
+      </div>
+      <div class="column column-flex">
+        <img src="images/dress2.jpeg">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
+        <div class="text-content">
+          <h3>Free People<span> | Size 6</span></h3>
+        </div>
+      </div>
+      <div class="column">
+        <div class="text-content">
+          <h3>Yves Saint Lauren<span> | One-Size</span></h3>
+        </div>
+      </div>
+      <div class="column">
+        <div class="text-content">
+          <h3>Tiffany<span> | One-Size</span></h3>
+        </div>
+      </div>
+      <div class="column">
+        <div class="text-content">
+          <h3>Louis Vuitton<span> | One Size</span>
+          </h3>
+        </div>
+      </div>
+      <div class="column">
+        <div class="text-content">
+          <h3>Reformation<span> | Size 4</span></h3>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column">
+        <div class="price">
+          <p>$35</p>
+          <span><del>$125 retail</del></span>
+        </div>
+      </div>
+      <div class="column">
+        <div class="price">
+          <p>$200</p>
+          <span><del>$1600 retail</del></span>
+        </div>
+      </div>
+      <div class="column">
+        <div class="price">
+          <p>$476</p>
+          <span><del>$200 retail</del></span>
+        </div>
+      </div>
+      <div class="column">
+        <div class="price">
+          <p>$95</p>
+          <span><del>$1200 retail</del></span>
+        </div>
+      </div>
+      <div class="column">
+        <div class="price">
+          <p>$20</p>
+          <span><del>$198 retail</del></span>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
While I can see why you broke these up into separate rows, the image and text content for each item are related information, so you should keep them all together. This will be a row of 5 columns, each of which should hold the image and all the text related to that product.